Dockerfile-CPU

```diff
@@ -20,11 +20,11 @@ RUN python3.9 -c "from setup import requirements; file = open('requirements.txt'
 COPY . /GaNDLF
 WORKDIR /GaNDLF
 RUN python3.9 -m pip install -e .
-# Entrypoint forces all commands given via "docker run" to go through python, CMD forces the default entrypoint script argument to be gandlf_run
-# If a user calls "docker run gandlf:[tag] gandlf_anonymize", it will resolve to running "python gandlf_anonymize" instead.
+# Entrypoint forces all commands given via "docker run" to go through python, CMD forces the default entrypoint script argument to be gandlf run
+# If a user calls "docker run gandlf:[tag] anonymize", it will resolve to running "gandlf anonymize" instead.
 # CMD is inherently overridden by args to "docker run", entrypoint is constant.
-ENTRYPOINT python3.9
-CMD gandlf_run
+ENTRYPOINT gandlf
+CMD run
```

Comment:
Dockerfiles are AFAIK the only place where this PR is temporarily broken.

docker images were built in the way when __image entrypoint__ (`python3.9`) is constant, while user can alter __CMD__. That allows to call various gandlf commands with the same image: entrypoint is `python3.9` and CMD includes the name of script user wants to run.

We moved `./gandlf_*` scripts to entrypoints. So right now they should be called as `gandlf_collectStats --foo bar` and cannot be called as `python gandlf_collectStats --foo bar`. Script name is __entrypoint__ now, not a part of __CMD__. So, the docker images are broken.

However, after we merge all the sub-PRs, everything will be fixed back: we'd use `gandlf` cli command as __entrypoint__, and subcommands (`run`, `collect-csv`, `debug-info`, ...) as __CMD parts__.

So, this fix is a bit "in advance": it would work only when subcommands are merged.